Formula/arangodb.rb

```diff
@@ -51,18 +51,27 @@ def install
       system "go", "build", *std_go_args(ldflags: ldflags), "github.com/arangodb-helper/arangodb"
     end
 
+    arch = case Hardware.oldest_cpu
+    when :arm_vortex_tempest
+      "neon"
+    when :core2
+      "merom" # only supports `core`, `merom`, and `penryn`
```

Comment:
Previous attempt at building older Core2 failed (#99988) so may not be well supported. May end up having to compile with at least `nehalem` again.

Reply:
Works now. Using `merom` to enable SSSE3 as `core` only allows up to SSE3.